docker/packager/binary/Dockerfile

```diff
@@ -97,7 +98,7 @@ ENV PATH="$PATH:/usr/local/go/bin"
 ENV GOPATH=/workdir/go
 ENV GOCACHE=/workdir/
 
-ARG CLANG_TIDY_SHA1=03644275e794b0587849bfc2ec6123d5ae0bdb1c
+ARG CLANG_TIDY_SHA1=c191254ea00d47ade11d7170ef82fe038c213774
```

Comment:
This is the version SHA of the clang-tidy-cache, right? Wondering why the existing one doesn't work with sccache?

Reply:
It's for https://github.com/matus-chochlik/ctcache/issues/17